dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -115,6 +116,21 @@ public static void main(String[] args) throws Exception {
         new DockstoreWebserviceApplication().run(args);
     }
 
+    public static String yaml2json(String yaml) {
```

Comment:
This probably belongs in a utility class rather than the main application class.